src/Search/Search.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import './Search.css';
+import propTypes from 'prop-types';
+
+class Search extends Component {
+  constructor(){
+    super()
+    this.state = {
+      city: ''
+    }
+  }
+
+  handleChange = (event) => {
+    this.setState({ city: event.target.value })
+  } 
+
+  render() {
+    return (
+      <div className="search-bar">
+        <div className="form-container">
+          <h2>Search by City:</h2>
+          <form className="form">
+            <input type="text" name="city" className="city-inp" placeholder="City" onChange={(event) => {
+              event.preventDefault();
+              this.handleChange(event);
+              this.props.filterPlaces(this.state.city);
+            }
```

Comment:
Refactor idea for a small cleanup. Lines 24-26 could be grouped into a single function outside the render, and then that function would be called here on line 23.

Reply:
Yeah I agree, we could do that when we refactor our code.